src/tools/management/commands/update_profile_picture.py

```diff
@@ -0,0 +1,52 @@
+from time import sleep
+
+from api.models import Profile
+from api.utils.auth0utils import get_user_info
+from .progress_bar_base_command import ProgressBarBaseCommand
+
+
+class Command(ProgressBarBaseCommand):
+    help = "Update profile picture from Auth0 profile"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--throttle",
+            type=int,
+            default=5,
+            help="Number of auth0 profiles to fetch before 1 second break.",
+        )
+
+        parser.add_argument(
+            "--profile_id",
+            type=str,
+            help="Only update profile for given profile id",
+        )
+
+    def update_profile_picture(self, profile):
+        auth_user = profile.authusers.all()[0]
```

Comment:
What if the first one doesn't have a `picture`? Should we iterate over `profile.authusers.all()` until we find one?